config/routes.rb

```diff
@@ -1,3 +1,19 @@
 Rails.application.routes.draw do
+  # get '/invite', to: 'invitations/new'
+  get 'invitations/show'
+  get '/invite', to: 'invitations#new'
+  get 'events/show'
+  get 'events/index'
+  get '/signup', to: 'users#new'
+  post '/signup', to: 'users#create'
+  get '/login', to: 'sessions#new'
+  post '/login', to: 'sessions#create'
+  delete '/logout', to: 'sessions#destroy'
+  get 'users/show'
+  get '/users', to: 'users#index'
```

Comment:
This is unnecessary because it's what you get by default. Remove this line and check `$ rails routes | grep user` and see the output. You'll notice that `/users` is already mapped to `users#index`.

[`REQUIRED`]